packages/wrangler/src/pages/deployment-tails.tsx

```diff
@@ -137,34 +138,54 @@ export async function Handler({
 		PAGES_CONFIG_CACHE_FILENAME
 	);
 	const accountId = await requireAuth(pagesConfig);
+	let deploymentId = deployment;
 
 	projectName = projectName || pagesConfig.project_name;
 
-	if (!deploymentId && !projectName && isInteractive) {
+	// When not specifying a deployment, we need a project name to use the latest
+	// Or when specifying by deployment URL
+	if ((!deployment || isUrl(deployment)) && !projectName && isInteractive) {
 		projectName = await promptSelectProject({ accountId });
 	}
 
-	if (!deploymentId && !projectName) {
-		throw new FatalError("Must specify a project name or deployment ID.", 1);
+	if (!deployment && !projectName) {
```

Comment:
shouldn't this throw when `deployment && !projectName`? Running `wrangler pages deployment tail <deployment>` returns 👇 . IMHO we should throw here and not allow an invalid request to go through

<img width="1081" alt="Screenshot 2022-11-11 at 12 14 43" src="https://user-images.githubusercontent.com/4638332/201329258-0d9db32b-5692-45d6-b40e-598d9a16c10a.png">